### ISSUE TITLE
Java 11, on clicking "edit overlays": class impl.org.controlsfx.version.VersionChecker cannot access class com.sun.javafx.runtime.VersionInfo

### ISSUE THREAD

**Issue by oskarkk** [Querz/mcaselector#185]
**Describe the bug**
When clicking "edit overlays" in menu, nothing happens. In console there's an error:

```
Exception in thread "JavaFX Application Thread" java.lang.IllegalAccessError: class impl.org.controlsfx.version.VersionChecker (in unnamed module @0x4e631548) cannot access class com.sun.javafx.runtime.VersionInfo (in module javafx.base) because module javafx.base does not export com.sun.javafx.runtime to unnamed module @0x4e631548
        at impl.org.controlsfx.version.VersionChecker.<clinit>(VersionChecker.java:50)
        at org.controlsfx.control.ControlsFXControl.<init>(ControlsFXControl.java:35)
        at org.controlsfx.control.RangeSlider.<init>(RangeSlider.java:175)
        at net.querz.mcaselector.ui.HueRangeSlider.<init>(HueRangeSlider.java:20)
        at net.querz.mcaselector.ui.OverlayBox.<init>(OverlayBox.java:87)
        at net.querz.mcaselector.ui.dialog.OverlayEditorDialog.add(OverlayEditorDialog.java:83)
        at net.querz.mcaselector.ui.dialog.OverlayEditorDialog.<init>(OverlayEditorDialog.java:52)
        at net.querz.mcaselector.ui.DialogHelper.editOverlays(DialogHelper.java:174)
        at net.querz.mcaselector.ui.OptionBar.lambda$new$28(OptionBar.java:131)
        at javafx.base/com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:86)
        at javafx.base/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:238)
        at javafx.base/com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:191)
        at javafx.base/com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
        at javafx.base/com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
        at javafx.base/com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
        at javafx.base/com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:49)
        at javafx.base/javafx.event.Event.fireEvent(Event.java:198)
        at javafx.controls/javafx.scene.control.MenuItem.fire(MenuItem.java:465)
        at javafx.controls/com.sun.javafx.scene.control.ContextMenuContent$MenuItemContainer.doSelect(ContextMenuContent.java:1380)
        at javafx.controls/com.sun.javafx.scene.control.ContextMenuContent$MenuItemContainer.lambda$createChildren$12(ContextMenuContent.java:1333)
(...)
```

**To Reproduce**
I've tried two Java versions. It doesn't happen with Java 8, but happens when I run mcaselector with Java 11 using this script:

```
#!/bin/sh

export PATH="/usr/lib/jvm/java-11-openjdk/bin/:$PATH"
exec java --module-path /usr/lib/jvm/java-11-openjfx/lib --add-modules ALL-MODULE-PATH -jar ~/software/mcaselector.jar
```
Maybe the bug arises from this particular setup, maybe I've overlooked something.

**Expected behavior**
Window with overlays should open.

**Environment (please complete the following information):**
- OS: Manjaro Linux
- Java version: jdk11-openjdk 11.0.10.u9-1, JavaFX version: java11-openjfx 11.0.10.u1-1
- Version 1.15

**Comment by Querz** [comment]
Apparently the controlsfx library is not backwards compatible, so the current version only runs with Java 8 but crashes with Java 9+ when trying to use the RangeSlider.

As a solution i made a (simplified) RangeSlider from scratch using only the public jfx API, which will be compatible with future Java versions and removed the controlsfx library as a dependency entirely.
This is going to be in a release later today.

People using the Windows installer are not affected by this, because it uses Java 8.

**Comment by Querz** [comment]
Fixed in release [1.15.1](https://github.com/Querz/mcaselector/releases/tag/1.15.1).

**Comment by oskarkk** [comment]
Great, thanks for your work!
